src/components/menu/Position/position.ts

```diff
@@ -0,0 +1,226 @@
+// (C) 2007-2018 GoodData Corporation
+
+export interface IDimensions {
+    width: number;
+    height: number;
+}
+
+export interface ICoordinates {
+    left: number;
+    top: number;
+    right: number;
+    bottom: number;
+}
+
+export interface IDimensionsAndCoordinates extends IDimensions, ICoordinates {}
+
+export type Direction = 'left' | 'right' | 'top' | 'bottom';
+
+export type Dimension = 'width' | 'height';
+
+export type IDirection =
```

Comment:
In the same file, there is `Direction` type and `IDirection` Despite almost the same name the type is different. In another file `IDirection` is exported as `Direction` which makes it all confusing. Could you adjust some of these names? For instance renaming `IDirection` to something like `IAlignment` ?

Reply:
Updated, used `Alignment` name for the type since it is not interface. Also renamed `direction` prop name to `alignment` to make it match closely with the type. Alignment also seem little more descriptive than direction in this case